Doc/c-api/tuple.rst

```diff
@@ -165,7 +165,7 @@ type.
 
    Describes a field of a struct sequence. As a struct sequence is modeled as a
    tuple, all fields are typed as :c:expr:`PyObject*`.  The index in the
-   :attr:`fields` array of the :c:type:`PyStructSequence_Desc` determines which
+   ``fields`` array of the :c:type:`PyStructSequence_Desc` determines which
```

Comment:
```suggestion
   :c:member:`~PyStructSequence_Desc.fields` array of the :c:type:`PyStructSequence_Desc` determines which
```
And replace the table above with definitions of `PyStructSequence_Desc` fields.

Reply:
I reverted this change.